docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -317,25 +317,50 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
+## Take over [filebeat-input-filestream-take-over]
+When `take_over` is enabled, this `filestream` input will take over
+states from the [`log`](/reference/filebeat/filebeat-input-log.md) input
+or other `filestream` inputs. Only states of files being actively
+harvested by this input are taken over.
 
-#### `take_over` [filebeat-input-filestream-take-over]
+To take over states from the log input, simply set `take_over.enabled:
+true`. To take over states from other `filestream` inputs, set
+`take_over.enabled: true` and set `take_over.from_ids` as list of
+previous `filestream` IDs you want to migrate states from.
```

Comment:
```suggestion
To take over files from a log input, simply set `take_over.enabled:
true` and add a matching path(s) to the `paths` list.

To take over states from other `filestream` inputs, set `take_over.enabled: true`, add matching paths to the `paths` list and set `take_over.from_ids` to a list of existing `filestream` IDs you want to migrate files from.

When `take_over.from_ids` is set, files are not taken over from `log` inputs. The migration is limited to `filestream` inputs only.
```

Reply:
Overall I like your suggestion, I just wonder if we really need to mention the paths. That's the most basic configuration of Filestream, paths has got to be set to the files the user wants ingested. It feels really redundant for me.

I like all other changes.

Reply:
I'm afraid that if we don't mention paths users would think it's enough to just set the `take_over.from_ids` and everything would magically work. From my standpoint it's better to be explicit about the complete requirements.

Reply:
I still believe it is redundant to mention that `paths` needs to be set. Specially as a step for the migration. I'll re-word it to mention the files we want to take over need to match the globs on `paths`.